unic/ucd/name/src/name.rs

```diff
@@ -74,21 +101,49 @@ impl Name {
     fn number_of_hex_digits(ch: char) -> usize {
         (32 - u32::leading_zeros(ch as u32) as usize + 3) / 4
     }
+
+    fn jamo_short_name(ch: char) -> &'static str {
+        data::JAMO_SHORT_NAMES
+            .find(ch)
+            .expect("Unexpected jamo character")
+    }
+
+    fn collect_jamo_short_names(ch: char) -> [&'static str; 3] {
+        let mut jamos = [""; 3];
```

Comment:
Curious if using an *empty string* is the best way of doing this here. Any thoughts?

Reply:
Changed the array to type `Option<&'static str>` with 3 elements so we can use `None` as the default values.
Feel like this makes more sense.